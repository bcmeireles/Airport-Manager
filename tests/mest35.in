a LIS a a
a POR a a
v TT5 LIS POR 14-02-2022 14:00 2:00 10

p LIS
e TT5
p LIS

q