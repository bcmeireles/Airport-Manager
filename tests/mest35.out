airport LIS
airport POR
TT5 POR 14-02-2022 14:00
